### DISCUSSION TITLE
Text input

### DISCUSSION THREAD

**Discussion by wilfredomerced** [twilio-labs/paste#1504] [General] [answered]
I had a question on the text area component. The input text appears to be visually different from whats in the DS docs. Is there a setting that I’m missing in figma?

**DS Docs**
<img width="805" alt="DS Docs" src="https://user-images.githubusercontent.com/81593493/119858849-146c8d00-bed2-11eb-934b-21d19c2c601b.png">
**Paste Component**
<img width="477" alt="figma component" src="https://user-images.githubusercontent.com/81593493/119858864-18001400-bed2-11eb-9392-024f35b5868e.png">

**Comment by gloriliale** [comment, marked as answer]
Hi @wilfredomerced! To clarify, the second screenshot is from the Paste Input Component when it's a text area in Figma?

**Reply by wilfredomerced** [reply]
yep yep

**Reply by gloriliale** [reply]
Is the styling difference you're talking about referring to the text inside of the text area? If so, the difference is from the docs having placeholder text and the figma component having text typed into it already. I might be missing something else entirely though, so please let me know if that's the case and we'll investigate!

**Reply by wilfredomerced** [reply]
so like when I just drop in the text input field component the placeholder text doesn't match whats in the docs
<img width="502" alt="Screen Shot 2021-05-27 at 10 56 04 AM" src="https://user-images.githubusercontent.com/81593493/119866667-45e95680-beda-11eb-9cfb-9e33f6a42e71.png">

**Reply by wilfredomerced** [reply]
sorry just found that you need to set the text component to be a placeholder. for designing purposes might be easier to start off as the text being set to placeholder by default because when you're designing screens you're going to design the first state of the input field

**Reply by serifluous** [reply]
~~Ah yeah, the Figma component shows the style of the typed value first since placeholder text isn't used too often.~~

~~If you want to swap to the placeholder style, you can click into the component in the layer list called ". Field text", and swap the variant to "Placeholder".~~

That's a good point. Placeholders aren't used too often in Twilio products, which is why we went with the typed value as the default. I've thought about having the form fields start off as blank...though that might make it harder for folks to figure out how to then add text to the form field. Very open to suggestions, though!

**Reply by wilfredomerced** [reply]
Hmm I wonder if you it could be easier to access in the panel of variants. its a bit confusing to access the variant for placeholder because I kept clicking on the text inside and not the text field part to change it to the placeholder text

**Reply by serifluous** [reply]
Yeah, hmm..we definitely want to take another pass at Buttons and Form fields, since they're our most variant-dense components. It might be cool to workshop it with a few PDs and decide on what variants are the most convenient to expose at the top level (hopefully without having to create hundreds of variants).

**Reply by wilfredomerced** [reply]
yeah I'd be down to participate
